snippets/snippets.json

```
@@ -24,5 +24,90 @@
 			"};",
 			""
 		]
+	},
+	"Require a @janiscommerce package": {
+		"scope": "javascript",
+		"prefix": "j-rqr",
+		"body": [
+			"const ${1:JanisModule} = require('@janiscommerce/${2:package-name}');",
+			"$0"
+		],
+		"description": "Require a @janiscommerce package"
+	},
+	"Create a model": {
+		"scope": "javascript",
+		"prefix": "model",
+		"body": [
+			"'use strict';",
+			"\nconst Model = require('@janiscommerce/model');",
+			"\nclass ${1:MyModel} extends Model {",
+			"${3:\n\tget databaseKey() {\n\t\treturn '$4';\n\t\\}}",
+			"${5:\n\tstatic get shouldCreateLogs() {\n\t\treturn false;\n\t\\}}",
+			"${6:\n\tstatic get uniqueIndexes() {\n\t\treturn [$7];\n\t\\}}",
+			"${8:\n\tstatic get table() {\n\t\treturn '$0';\n\t\\}}",
+			"}",
+			"\nmodule.exports = ${1:MyModel};\n"
```

```suggestion
			"\nmodule.exports = const Model = require('@janiscommerce/model');",
			"\nclass ${1:MyModel} extends Model {",
			"${3:\n\tget databaseKey() {\n\t\treturn '$4';\n\t\\}}",
			"${5:\n\tstatic get shouldCreateLogs() {\n\t\treturn false;\n\t\\}}",
			"${6:\n\tstatic get indexes() {\n\t\treturn [$7];\n\t\\}}",
			"${8:\n\tstatic get table() {\n\t\treturn '$0';\n\t\\}}",
			"};\n"
```

El getter de `uniqueIndexes` no va más.